pkg/construct2/graph_io.go

```diff
@@ -119,10 +124,20 @@ func (g *YamlGraph) UnmarshalYAML(n *yaml.Node) error {
 	}
 
 	var errs error
-	for rid, props := range y.Resouces {
+	for rid, props := range y.Resources {
+		var imported bool
+		if imp, ok := props["imported"]; ok {
+			val, ok := imp.(bool)
+			if !ok {
+				errs = errors.Join(errs, fmt.Errorf("unable to parse imported value as boolean for resource %s", rid))
```

Comment:
```suggestion
				errs = errors.Join(errs, fmt.Errorf("unable to parse imported value as boolean for resource %s", rid))
				// Don't continue here so that the vertex is still added, otherwise it could erroneously cause failures in the edge copying
```